### DISCUSSION TITLE
Why is XAML not recognized as a language?

### DISCUSSION THREAD

**Discussion by Schecher1** [community/community#21082] [Other Feature Feedback, Questions, & Ideas]
I think XAML should be recognized at repos as a language, for example if I have a software with C# and XAML, it will show only 100% C# and not 20% XAML or something else. In addition, I find XAML not even so unknown.

![image](https://user-images.githubusercontent.com/74829802/180481496-84d032d2-9e8f-42c2-981a-c8d78cab5ee3.png)

**XAML is not C#!**

**Comment by look** [comment]
This is determined by [Linguist](https://github.com/github/linguist). Reviewing the data file, I see the `.xaml` extension is marked as an alias of XML, so I would expect XAML files to be detected as XML.

If you have a public example where this isn't correct, please link to it.

**Reply by Schecher1** [reply]
ups, i accidentally created a new answer https://github.com/orgs/github-community/discussions/21082#discussioncomment-3208556

**Comment by Schecher1** [comment]
Here as an example, this program was written by me it is a WPF program, this has as FrontEnd language XAML, you only see C# 100% and nowhere XML or XAML. 

Apart from that, I think XML as a suggestion is confusing. But in the worst case I'll accept it.


![image](https://user-images.githubusercontent.com/74829802/180506422-d564ede3-26a7-43d9-ac23-3d561bf3d17e.png)

and e.g. here it is recognized as XML, but not on the "main" view

![image](https://user-images.githubusercontent.com/74829802/180506506-9e9b3a3a-dc70-4c5a-9e10-8d531741e03b.png)



https://github.com/Schecher1/Encrypted-Notebook-EntityFramework
